R/toplevel_util.R

```diff
@@ -9,7 +9,9 @@ row_as_header <- function(df, row_index = 1, prefix = "", clean_names = TRUE){
 
   loadNamespace("stringr")
   loadNamespace("janitor")
-  names <- as.character(df[row_index, ])
+  header_row <- df[row_index, ]
+  # make values to character
```

Comment:
Please add comment explaining why we do this rather than just as.character like before this change.